packages/deployments/contracts/contracts/messaging/connectors/SpokeConnector.sol

```diff
@@ -289,6 +257,8 @@ abstract contract SpokeConnector is Connector, ConnectorManager, WatcherClient,
    * @dev The root of this tree will eventually be dispatched to mainnet via `send`. On mainnet (the "hub"),
    * it will be combined into a single aggregate root by RootManager (along with outbound roots from other
    * chains). This aggregate root will be redistributed to all destination chains.
+   *
+   * NOTE: okay to leave dispatch operational when paused as pause is designed for crosschain interactions
```

Comment:
This `NOTE` contradicts the `whenNotPaused` modifier above lol

Reply:
kek

Reply:
wait, dispatch isn't `whenNotPaused`, but `send()` is -- i.e. you can't communicate between chains but you can add messages to the outbox

Reply:
[L261-267](https://github.com/connext/nxtp/blob/360cc4ef125d2bd49ca52e1eb70d9ca75054c9a7/packages/deployments/contracts/contracts/messaging/connectors/SpokeConnector.sol#L261-L267)
```
   * NOTE: okay to leave dispatch operational when paused as pause is designed for crosschain interactions
   */
  function dispatch(
    uint32 _destinationDomain,
    bytes32 _recipientAddress,
    bytes memory _messageBody
  ) external onlyWhitelistedSender returns (bytes32) {
  ```